hardware/OnkyoAVTCP.cpp

```diff
@@ -364,7 +546,7 @@ void OnkyoAVTCP::ParseData(const unsigned char *pData, int Len)
 			continue;
 		}
 		int data_size = static_cast<int>(ntohl(pkt->data_size));
-//		_log.Log(LOG_NORM, "ISCP at len %d size %d", Len, data_size);
+		_log.Log(LOG_NORM, "ISCP at len %d size %d", Len, data_size);
```

Comment:
I think this is a debug log and should be removed?

Reply:
Removed.